ext/intl/intl_convertcpp.cpp

```diff
@@ -79,8 +79,8 @@ zend_string* intl_charFromString(const UnicodeString &from, UErrorCode *status)
 		zend_string_free(u8res);
 		return NULL;
 	}
+	u8res = zend_string_truncate(u8res, actual_len, 0);
 	ZSTR_VAL(u8res)[actual_len] = '\0';
```

Comment:
Is this line also needed? I would expect the ``zend_string_truncate()`` function to provide a null terminated string.

Reply:
I guess so :)